sdk/monitor/azure-monitor-opentelemetry-exporter/README.md

```diff
@@ -58,7 +58,7 @@ SpanExporter exporter = new AzureMonitorExporterBuilder()
     .buildTraceExporter();
 
 SdkTracerProvider tracerProvider = SdkTracerProvider.builder()
-    .addSpanProcessor(SimpleSpanProcessor.create(exporter))
+    .addSpanProcessor(BatchSpanProcessor.builder(exporter).build())
```

Comment:
is there a chance that spans won't be flushed when the app stops (if someone runs the example as is)?